components/section-slideshow.jsx

```
@@ -0,0 +1,139 @@
+  const [activeIndex, setActiveIndex] = useState(0);
+
+  // Kiểm tra và lọc các hình ảnh hợp lệ
+  const validImages = images?.filter(img => {
+    if (!img) return false;
+
+    // Kiểm tra xem hình ảnh có URL hay không
+    if (img.url) return true;
+
+    // Nếu không có URL trực tiếp, kiểm tra xem có asset hay không
+    if (img.asset || (img.image && img.image.asset)) {
+      console.log('Image with asset but no URL in section slideshow:', img);
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)